server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/MailboxManageTest.java

```diff
@@ -149,16 +184,28 @@ void mailboxListWithTwoAddedMailboxesAndExistedUsernameShouldShowMailboxesNameEx
             .addUser("hqtran@linagora.com", "123456");
 
         WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
-            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.1");
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX1");
 
         WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
-            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.2");
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX2");
 
         int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
             "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "list", "hqtran@linagora.com");
```

Comment:
test name says you test exist command... but this is list?

Reply:
this test's name is mailboxListWithTwoAddedMailboxesAndExistedUsernameShouldShowMailboxesNameExactly() (line 182)

Reply:
You are right, i didn't see the fold of code in the middle sorry